setup.cfg

```diff
@@ -23,6 +23,7 @@ install_requires =
     pyyaml
     intervaltree
     hissw
+    polling2
```

Comment:
I guess I'm a little bit concerned about having a dependancy on a one man library

Reply:
yes but this is a very isolated function and location in the code
if problems show up we can replace it